src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -558,6 +563,13 @@ private NativeSearchQueryBuilder newSearchQuery(BoolQueryBuilder branchUpdatesCr
 				.withPageable(LARGE_PAGE);
 	}
 
+	private QueryBuilder getNonStatedModifiedReleasedRelationshipClause() {
+		return boolQuery()
+				.must(existsQuery(Relationship.Fields.EFFECTIVE_TIME))
+				.mustNot(existsQuery(Relationship.Fields.RELEASED))
+				.mustNot(termsQuery(Relationship.Fields.CHARACTERISTIC_TYPE_ID, Concepts.INFERRED_RELATIONSHIP, Concepts.ADDITIONAL_RELATIONSHIP));
+	}
+
```

Comment:
When writing a unit test, I learned that my Relationships (inferred relationships to mimic released) were not being found in the merge. I dug through and found it was from the `getNonStatedRelationshipClause` method explicitly ignoring inferred Relationships. 

I have therefore created `getNonStatedModifiedReleasedRelationshipClause` to limit the scope of my change, but I think this is correct: find Relationships that are released, but also modified.